.changeset/config.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@3.0.0/schema.json",
+  "changelog": [
+    "@changesets/changelog-github",
+    { "repo": "bigcommerce/catalyst" }
+  ],
+  "commit": false,
```

Comment:
Assumption: By default, Changesets should not write the commit messages for version bumps. We instead want the author of the PR that results in a release to leave a descriptive commit message for the work done in their PR.